packages/cli/src/commands/generate/cell/templates/cellList.js.template

```diff
@@ -0,0 +1,27 @@
+export const QUERY = gql`
+  query ${operationName} {
+    ${camelName} {
+      id
+    }
+  }
+`
+
+export const Loading = () => <div>Loading...</div>
+
+export const Empty = () => <div>Empty</div>
+
+export const Failure = ({ error }) => <div style={{color: 'red'}}>Error: {error.message}</div>
+
+export const Success = ({ ${camelName} }) => {
+  return (
+    <ul>
+      {${camelName}.map((item) => {
+        return (
+          <li key={item.id}>
+            <div>{JSON.stringify(item)}</div>
```

Comment:
love that we're rendering the array as a list now. since it's already a list item, do we need the div here?
```suggestion
            {JSON.stringify(item)}
```

Reply:
No we don't I think! Will take out